### PR TITLE
docs: Update getting_started.md

### PR DESCRIPTION
**Pull request by patrinoua**
The type of this PR is: **docs**

This PR updates the documentation

### PR Checklist (tick all before merging)

- [ ] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [ ] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ yarn install:all
 
 **Note**: `bundle exec pod install` may fail the first time you run it (due to a [bug](https://github.com/orta/cocoapods-keys/issues/127) in a dependency of ours). Re-running the command should work.
 
+## Find a ticket 
+If you're just starting you can find a ticket on the [CX backlog](https://artsyproduct.atlassian.net/jira/software/c/projects/CX/boards/77/backlog?issueLimit=100)
```
[the remark this answers is not in the document]
Comment (patrinoua) [reply docs/getting_started.md:57]: I'm assuming that mainly Artsy people work on eigen, therefore I thought it would nice to have a link here, but maybe I'm wrong, do we get PRs from independent contributors often? 
Also if I'm not mistaken Jira is closed for external people so there should not be a privacy issue. 😊

[the remark this answers is not in the document]
Comment (pvinis) [reply docs/getting_started.md:57]: All true, but I would be put off if some open source project had "Find a ticket" link to a private jira, and we are trying to keep eigen friendly to independent contributors 😬.

How about this, we rephrase this to something like "If you work at Artsy, find a ticket on <jira link>. If you are not working at Artsy, make a ticket if you need, or find a ticket from <link to issues>, or contribute anything else you have an idea for."? 🤔

**Review by pvinis — APPROVED** [review]
